### DISCUSSION TITLE
Support all `cursor-` utilities in JIT mode

### DISCUSSION THREAD

**Discussion by targos** [tailwindlabs/tailwindcss#5724] [Ideas]
Currently, the list of available `cursor-` utilities depends on the configuration. It's okay and relatively easy to add more to the list (for example, I needed `cursor-grab`).
Would it be possible to support all [cursor](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor) values by default in JIT mode?

**Comment by adamwathan** [comment]
Hey! Great idea, happy to accept a PR for this if you want or can work on it when I find some time 👍🏻 

Basically just need to add the options here:

https://github.com/tailwindlabs/tailwindcss/blob/master/stubs/defaultConfig.stub.js#L232-L241

**Comment by dance2die** [comment]
@targos Can I take on the PR if you aren't working on it?  

---

Working on the changes (not PRed), https://github.com/tailwindlabs/tailwindcss/compare/master...dance2die:master

---

@adamwathan: The [CONTRIBUTING.md](https://github.com/tailwindlabs/tailwindcss/blob/master/.github/CONTRIBUTING.md) seems out of date because 

1. there is no `babelify` scripts in package.json.
2. `npm run prepublishOnly` doesn't produce `dist/`

**Reply by targos** [reply]
@dance2die sure, go ahead!

**Reply by dance2die** [reply]
Thank you, @targos.  
PR opened https://github.com/tailwindlabs/tailwindcss/pull/5734

**Reply by simonswiss** [reply]
Nice to see this merged – thank you! 🎉
